HealthVault/Classes/Types/MHVPersonInfo.h

```diff
@@ -20,12 +20,13 @@
 #import "MHVBaseTypes.h"
 #import "MHVType.h"
 #import "MHVRecord.h"
+#import "MHVApplicationSettings.h"
 
 @interface MHVPersonInfo : MHVType
 
 @property (readwrite, nonatomic, strong) NSUUID *ID;
 @property (readwrite, nonatomic, strong) NSString *name;
-@property (readwrite, nonatomic, strong) NSString *appSettingsXml;
+@property (readwrite, nonatomic, strong) MHVApplicationSettings *applicationSettings;
```

Comment:
Yeah, I can make it pass-through to MHVApplicationSettings  internally, to make it simpler for the SDK users